### PR TITLE
Add power sample

### PR DESCRIPTION
**Pull request by joel16**
This is untested at the moment. I don't own a vita (yet). Mine should arrive by tomorrow or the day after. Please test and let me know if it works fine before merging. It compiles without any problems.

### CODE REVIEW

File: power/src/main.c [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	printf("Battery life time: (%02dh%02dm)\n", batteryLifeTime/60, batteryLifeTime-(batteryLifeTime/60*60));
+	printf("Clock frequency of the ARM: %d mHz\n", scePowerGetArmClockFrequency());
+	printf("Clock frequency of the BUS: %d mHz\n", scePowerGetBusClockFrequency());
+	printf("Clock frequency of the GPU: %d mHz\n", scePowerGetGpuClockFrequency());
+	
+	printf("\nPress X to set ARM between 444 and 333 mHz\n");
+
+	
+	//check to see if the user pressed X to toggle CPU speed
+	if (pad.buttons & SCE_CTRL_CROSS)
```
Comment (xyzz) [inline power/src/main.c]: definitely not, considering `pad` is never updated before here. Also, the indentation is all over the place, fix it please.

**Comment by joel16** [comment]
My indentation isn't bad. It's a problem with github. On my notepad++ it shows up just fine, but sometimes when I push to github it messes with indentation. However once the source is downloaded it's displayed properly.  Anyways I'll fix the pad stuff once I get my Vita. I'll have some time during the weekend.

**Comment by joel16** [comment]
Alright indentation issues should be fixed now. And pad also should be updated.

**Comment by joel16** [comment]
Actually I'll make some more changes to this later, and submit a new pull request with just one change.

**Pull request closed without merging**